src/Adapter/MySQL.php

```diff
@@ -320,7 +321,9 @@ private function computeOrderByField(array $filterToTableMapping)
         if ($orderField === 'price') {
             $orderField = $this->getOrderDirection() === 'asc' ? 'price_min' : 'price_max';
         }
-
+        
+        // COULD BE CHANGED WITH METHOD computeFieldName($fieldName)
+        // example: $orderField = $this->computeFieldName($orderField, $filterToTableMapping);
```

Comment:
it must be refactor to prevent duplicated code.

Reply:
Done. Commit: https://github.com/PrestaShop/ps_facetedsearch/pull/154/commits/21cc62cb3bc4b9688e2336f54293e93e083a2464